src/main/java/com/cairn/waypoint/dashboard/endpoints/homeworkquestion/GetHomeworkQuestionByCategoryEndpoint.java

```diff
@@ -0,0 +1,92 @@
+package com.cairn.waypoint.dashboard.endpoints.homeworkquestion;
+import com.cairn.waypoint.dashboard.endpoints.ErrorMessage;
+import com.cairn.waypoint.dashboard.endpoints.homework.dto.HomeworkListDto;
+import com.cairn.waypoint.dashboard.endpoints.homework.dto.HomeworkQuestionListDto;
+import com.cairn.waypoint.dashboard.endpoints.homeworktemplate.dto.HomeworkQuestionDetailsListDto;
+import com.cairn.waypoint.dashboard.service.data.HomeworkDataService;
+import com.cairn.waypoint.dashboard.service.data.HomeworkQuestionDataService;
+import com.cairn.waypoint.dashboard.service.data.HouseholdDataService;
+import com.cairn.waypoint.dashboard.service.helper.HomeworkHelperService;
+import com.cairn.waypoint.dashboard.service.helper.HomeworkQuestionHelperService;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.security.SecurityRequirement;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.security.Principal;
+import java.time.LocalDateTime;
+import java.util.stream.Collectors;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController
+@Tag(name = "Homework")
+public class GetHomeworkQuestionByCategoryEndpoint {
+	
+	  public static final String PATH = "/api/homework-question/category/{categoryId}";
+
+	  private final HomeworkQuestionDataService homeworkQuestionDataService;
+	  private final HomeworkQuestionHelperService homeworkQuestionHelperService;
+	 
+	  public GetHomeworkQuestionByCategoryEndpoint( HomeworkQuestionHelperService homeworkQuestionHelperService, HomeworkQuestionDataService homeworkQuestionDataService) {
+		this.homeworkQuestionDataService = homeworkQuestionDataService;
+		this.homeworkQuestionHelperService = homeworkQuestionHelperService;
+	  }
+
+
+@GetMapping(PATH)
```

Comment:
Check your spacing/indenting here. This is off. We can prioritize the style guide next meeting